### PR TITLE
Small fixes

### LINKED ISSUE #5: Initial thoughts on required changes
I'm opening this issue to gather my thoughts, feel free to split this into different issues:
1. Release flow is needed along side a changelog, we might be able to automate it.
2. `style-spec.ts` should be renamed `index.ts`
3. `types.g.ts` should probably be generated and not checked-in`?
4. I would consider moving some files to internal folders and keep only `index.ts` in the top level `src` folder?
5. Add code coverage?
6. Improve readme - add link to npm, latest version? number of installations? badges? maplibre icon?
7. ...

### LINKED ISSUE #236: Create contributor guide for maplibre-style-spec
There is currently no contributor guide for maplibre-style-spec. A contributor guide should cover the steps needed to establish a development environment where a contributor can develop and test a PR which has changes in both maplibre-style-spec and gl-js (or native). I've tried `npm link` and `npm install <local folder>` approaches to set up a multi-project environment and have failed after multiples hours of attempts and much debugging.  I would happily offer to write up steps if I could figure it out myself :grimacing:

### LINKED ISSUE #458: Two different SDK support tables for one item
**Describe the bug**
[Raster DEM source volatile attribute](https://maplibre.org/maplibre-style-spec/sources/#raster-dem-volatile) has two different SDK support tables:

image

### PR DESCRIPTION
**Pull request by HarelM**
This fixes some small issues:
Fixes #458, Fixes #5, Fixes #236
- #458
- #5
- #236

Also fixes a bug on mobile where you can unintentionally scroll the screen sideways.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+It is a good idea to discuss proposed changes before proceeding to an issue ticket or PR. The project team is active in the following forums:
+
+* For informal chat discussions, visit the project's [Slack Channel](https://osmus.slack.com/archives/C01G3D28DAB).
+* For discussions whose output and outcomes should not be ephemeral, consider starting a thread on [GitHub Discussions](https://github.com/maplibre/maplibre-style-spec/discussions). This makes it easier to find and reference the discussion in the future.
+
+MapLibre software relies heavily on automated testing, and the project includes a suite of unit and integration tests. For both new features and bug fixes, contributions should update or add test cases to prevent regressions.
+
+# Building the Docs site
+
+The MapLibre style spec documnetions site is based on [SOLIDSTART](https://start.solidjs.com/) in order to get Server Side Rendering for Search Engnie Optimization.
```
Comment (louwers) [inline CONTRIBUTING.md:20]:
```suggestion
The MapLibre style spec documnetions site is based on [SolidStart](https://start.solidjs.com/) in order to get Server Side Rendering for Search Engnie Optimization.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+It also has some other tools as can be read in the main README file.
+
+```bash
+npm install
+npm run build
+```
+
+# Running the tests
+
+Unit tests are using jest.
```
Comment (louwers) [inline CONTRIBUTING.md:40]:
```suggestion
Unit tests are using Jest.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+Hi, and thanks in advance for contributing to MapLibre GL JS. Here's how we work. Please follow these conventions when submitting an issue or pull request.
```
Comment (louwers) [inline CONTRIBUTING.md:1]:
```suggestion
Hi, and thanks in advance for contributing to the MapLibre Style Spec. Here's how we work. Please follow these conventions when submitting an issue or pull request.
```

File: docs/src/root.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ export default function Root() {
             <Head>
                 <Title>MapLibre Style Spec</Title>
                 <Meta charset="utf-8" />
-                <Meta name="viewport" content="width=device-width, initial-scale=1" />
+                <Meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no, viewport-fit=cover" />
```
Comment (louwers) [inline docs/src/root.tsx:25]: > Warning: Usage of user-scalable=no can cause accessibility issues to users with visual impairments such as low vision. [WCAG](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background) requires a minimum of 2× scaling; however, the best practice is to enable a 5× zoom.

https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#viewport_basics

This is not ideal for accessibility.

**Review by louwers — APPROVED** [review]
Looks good. I did not try to build it myself.

Feel free to make an issue for the accesibility note I made, and I can have a look at it later.

**Comment by louwers** [comment]
@HarelM Can disable auto-merge when there are open conversations? I didn't mean to merge this yet.

**Comment by HarelM** [comment]
My bad... Can you open a PR with the changes? Would be easier I think...

File: docs/src/root.tsx (hunk 4 above)
Comment (HarelM) [reply docs/src/root.tsx:25]: Please do, yes, this is the solution I'm familiar with...

**Comment by HarelM** [comment]
I'll add it to my other PR...

**Comment by louwers** [comment]
@HarelM Thanks!

**Pull request merged**